content/en/docs/cheat-sheet/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Cheat sheet"
+images: []
+menu:
+docs:
+weight: 100
+---
```

Comment:
A `Tips` page might be added later to the cheat sheet section, providing tips on questions such as `Which Introspector should I use?`